src/index.js

```diff
@@ -1,13 +1,10 @@
 // @flow
 
 import * as React from 'react'
-import { AppRegistry, YellowBox } from 'react-native'
+import { AppRegistry } from 'react-native'
 import App from './modules/app/components/App'
 import 'moment/min/locales'
 
-// @see: https://github.com/facebook/metro/issues/287
-YellowBox.ignoreWarnings(['Require cycle:'])
-
```

Comment:
The issue is not resolved (or is it?). Why did you remove this?

Reply:
I think we should keep an eye on this. Also the API of YellowBox changed so not sure whether this still works.

I think most libraries just need an update to remove this warning. at least I did not see it anymore.